src/transport/tcp/libxev.zig

```diff
@@ -0,0 +1,693 @@
+const std = @import("std");
+const xev = @import("xev");
+const Intrusive = @import("../../queue_mpsc.zig").Intrusive;
+const IOQueue = Intrusive(AsyncIOQueueNode);
+const TCP = xev.TCP;
+const Allocator = std.mem.Allocator;
+const ThreadPool = xev.ThreadPool;
+const ResetEvent = std.Thread.ResetEvent;
+
+/// SocketChannel represents a socket channel. It is used to send and receive messages.
+pub const SocketChannel = struct {
+    /// The underlying socket.
+    socket: TCP,
+
+    /// The transport that created this channel.
+    transport: *XevTransport,
+
+    /// Whether this channel is the initiator of the connection. If true, this channel is the client. If false, this channel is the server.
+    is_initiator: bool,
```

Comment:
may be we have an enum `Inbound`/`OutBound` here to make it clear implicitly

Reply:
Yes, i also thought this, will change it